src/packet/ipv4.rs.in

```diff
@@ -14,6 +14,18 @@ use pnet_macros_support::types::*;
 
 use std::net::Ipv4Addr;
 
+/// IPv4 header flags
+#[allow(non_snake_case)]
+#[allow(non_upper_case_globals)]
+pub mod Ipv4Flags {
+    use pnet_macros_support::types::*;
+
+    /// Don't Fragment flag
+    pub const DF: u3 = 0b010;
```

Comment:
Could these be `DontFragment` and `MoreFragments`? It's only a few extra characters, and makes it a lot more readable.